domains/baodev.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "gibaohung-dev",
+        "email": "thisismegiabao@gmail.com"
+    },
+    "record": {
+        "URL": "hosts.is-a.dev"
```

Comment:
is-a.dev Hosting was shutdown.

Reply:
so what can i do?

Reply:
Use GitHub Pages or another website hosting service to host a website on an is-a.dev subdomain.